WordPress/Classes/ViewRelated/Post/PostEditor+MoreOptions.swift

```diff
@@ -14,10 +14,31 @@ extension PostEditor where Self: UIViewController {
         self.navigationController?.pushViewController(settingsViewController, animated: true)
     }
 
+    private func savePostBeforePreview(completion: @escaping (() -> Void)){
+        let context = ContextManager.sharedInstance().mainContext
+        let postService = PostService(managedObjectContext: context)
+        if post.isDraft() {
```

Comment:
@aerych , if I add a check for hasUnsavedChanges and preview the post for the second time without making changes it will show the local preview. What do you think we should do here?